pkg/reconciler/instances/ory/hydra/syncer_test.go

```diff
@@ -62,51 +63,48 @@ func Test_TriggerSynchronization(t *testing.T) {
 
 		// then
 		require.NoError(t, err)
-		require.NoError(t, err)
 		kubeclient.AssertNotCalled(t, "PatchUsingStrategy", mock.Anything, mock.Anything, mock.Anything, mock.Anything, mock.Anything, mock.Anything)
 	})
-	t.Run("Should  trigger synchronization when hydra is behind hydra-maester but sync forced", func(t *testing.T) {
+
+	t.Run("Should trigger synchronization when earliest start times of hydra is equal to hydra-maester", func(t *testing.T) {
 		// given
-		hydraStartTimePod1 := time.Date(2021, 10, 10, 10, 10, 10, 10, time.UTC)
-		hydraStartTimePod2 := time.Date(2021, 10, 10, 10, 10, 7, 10, time.UTC)
-		hydraStartTimePod3 := time.Date(2500, 10, 10, 10, 10, 6, 10, time.UTC)
+		hydraStartTimePod := time.Date(2022, 10, 10, 10, 10, 7, 10, time.UTC)
+		hydraStartTimePod2 := time.Date(2022, 10, 10, 10, 10, 6, 10, time.UTC)
 		hydraMasesterPodStartTime := time.Date(2022, 10, 10, 10, 10, 6, 10, time.UTC)
 		kubeclient := fakeClient()
-		addPod(kubeclient, "hydra1", "hydra", hydraStartTimePod1, t, v1.PodRunning)
+		addPod(kubeclient, "hydra1", "hydra", hydraStartTimePod, t, v1.PodRunning)
 		addPod(kubeclient, "hydra2", "hydra", hydraStartTimePod2, t, v1.PodRunning)
-		addPod(kubeclient, "hydra3", "hydra", hydraStartTimePod3, t, v1.PodPending)
 		createDeployment(kubeclient, hydraMasesterPodStartTime, t)
```

Comment:
maybe we can do some renaming to improve readability?
```suggestion
		hydraStartTimeEarlier := time.Date(2022, 10, 10, 10, 10, 7, 10, time.UTC)
		hydraStartTimeLater := hydraStartTimeEarlier.Add(time.Second)
		hydraMasesterPodStartTime := hydraStartTimeEarlier
		kubeclient := fakeClient()
		addPod(kubeclient, "hydra1", "hydra", hydraStartTimeEarlier, t, v1.PodRunning)
		addPod(kubeclient, "hydra2", "hydra", hydraStartTimeLater, t, v1.PodRunning)
```